packages/scripts.nix

```diff
@@ -91,6 +91,54 @@
       '';
   };
 
+  upload-release-image = writeShellApplication {
+    name = "upload-release-image";
+    runtimeInputs = with pkgs; [
+      azure-cli
+      uplosi
+    ];
+    text =
+      let
+        image = pkgs.image-podvm;
+      in
+      ''
+        imageVersion=""
+
+        for i in "$@"; do
+          case $i in
+          --version=*)
+            imageVersion="''${i#*=}"
+            shift
+            ;;
+          *)
+            echo "Unknown option $i"
+            exit 1
+            ;;
+          esac
+        done
+
+        set -x
+
+        # Create an uplosi config.
+        cat <<EOF > uplosi.conf
+        [base]
+        imageVersion = "''${imageVersion}"
+        name = "contrast"
+        provider = "azure"
+
+        [base.azure]
+        subscriptionID = "0d202bbb-4fa7-4af8-8125-58c269a05435"
+        location = "GermanyWestCentral"
+        resourceGroup = "contrast-images"
+        sharedImageGallery = "contrast_images"
+        sharingProfile = "community"
+        sharingNamePrefix = "Contrast"
```

Comment:
I'd recommend adding something like:
```sh
[variant.default.azure]
replicationRegions = [...]
```

And make sure that the other regions where potential customers are testing would also have the image uploaded to them.

Reply:
Good idea! I added the same regions we also use for [constellation](https://github.com/edgelesssys/constellation/blob/54058eed2a6895de2708d90485cf4e9daea3414a/internal/osimage/uplosi/uplosi.conf.in#L6).

Reply:
Why not use the [locations contrast supports](https://docs.edgeless.systems/contrast/getting-started/cluster-setup#create-resource-group)?